Strain_Tools/strain/models/strain_gpsgridder.py

```diff
@@ -77,16 +77,7 @@ def compute_gpsgridder(myVelfield, range_strain, inc, poisson, fd, eigenvalue, t
     xinc = xinc * 111.000 * np.cos(np.deg2rad(range_strain[2]));  # in km (not degrees)
     yinc = yinc * 111.000;   # in km (not degrees)
 
-    [exx, eyy, exy, rot] = strain_tensor_toolbox.strain_on_regular_grid(xinc, yinc, udata, vdata)
-    # Lastly we multiply by 1000 for units
-    exx = exx.T;
-    eyy = eyy.T;
-    exy = exy.T;
-    rot = rot.T;
-    exx = np.multiply(exx, 1000);
-    exy = np.multiply(exy, 1000);
-    eyy = np.multiply(eyy, 1000);
-    rot = abs(np.multiply(rot, 1000));
+    [exx, eyy, exy, rot] = strain_tensor_toolbox.strain_on_regular_grid(xinc, yinc, udata/1000, vdata/1000)
```

Comment:
@jlmaurer just tested this - the line should be (..., udata*1000, vdata*1000). Then I get the same answers. could you add that and then I'll merge?

Reply:
I'm adding a few changes (pygmt long-form-arguments, etc.) to my branch but there will be no conflicts

Reply:
@kmaterna done! Also fixed a bug I had in the geostats module with the dx calcualtion